certified-connectors/Asite/apiDefinition.swagger.json

```diff
@@ -765,5 +914,44 @@
       ]
     }
   ],
-  "tags": []
+  "tags": [],
+  "x-ms-capabilities": {
+    "file-picker": {
+      "open": {
+        "operationId": "GET_ASITE_FOLDER_LIST",
+        "parameters": {
+          "projectId": {
+            "parameter": "projectId"
+          },
+          "isFolderDataReq": {
+            "value": true
+          },
+          "isWorkspace": {
+            "value": 1
+          },
+          "Accept": {
+            "value": "*/*"
+          }
+        }
+      },
+      "browse": {
+        "operationId": "GET_ASITE_SUB_FOLDER_LIST",
+        "parameters": {
+          "projectId": {
+            "parameter": "projectId"
+          },
+          "folderId": {
+            "value-property": "folderId"
+          },
+          "hasSubFolder": {
+            "value": true
+          }
+        }
+      },
+      "value-title": "folder_title",
+      "value-path": "folderId",
+      "value-media-property": "",
```

Comment:
@asitedeveloper It is intentional to keep empty string or a typo. If it is not needed, please remove it.

Reply:
@sriyen-msft I have changed the value of media property.